orttraining/orttraining/core/graph/gradient_builder.cc

```diff
@@ -1699,16 +1700,17 @@ IMPLEMENT_GRADIENT_BUILDER(GetATenOpGradient) {
   std::vector<ArgDef> output_args;
 
   for (const auto& config : op_config.backward_input_source_configs) {
-    size_t index = config.second;
-    switch (config.first) {
+    size_t index = std::get<1>(config);
```

Comment:
shall we make backward_input_source_configs a named struct?
looking for the right field in std::tuple is less intuitive...